life_server/Functions/Actions/fn_pickupAction.sqf

```diff
@@ -2,23 +2,22 @@
 /*
     File: fn_pickupAction.sqf
     Author: Bryan "Tonic" Boardwine
-
     Description:
     Validates that the cash is not a lie
 */
-params [
-    ["_obj",objNull,[objNull]],
-    ["_client",objNull,[objNull]],
-    ["_cash",false,[true]]
-];
+private _target = param [0, objNull, [objNull]];
+
+if (!isRemoteExecuted) exitWith { ["Not remote executed"] call BIS_fnc_error };
+if (isNull _obj) exitWith { ["Target object is NULL"] call BIS_fnc_error };
```

Comment:
don't know if there's a real reason, but I don't think Altis Life uses this function, so we shouldn't start doing so now for consistencies sake